post-scan-actions/aws-python-email-notification/README.md

```diff
@@ -0,0 +1,262 @@
+        - Select its check box in the list.
+        - You now have two policies selected.
+        - Click **Next: Tags**.
+        - (Optional) Enter tags.
+        - Click **Next: Review**.
+    6. On the **Review** page:
+        - In the **Role name** field, enter a name. Example: `FSS_Lambda_Email_Role`.
+        - Make sure that two policies are listed.
+        - Click **Create role**.
+    </details>
+
+   <details>
+   <summary>Using the AWS CLI</summary>
+
+    1. Enter the following AWS CLI command to create the role:
+
+        `
+        LAMBDA_TRUST="{
+            \"Version\": \"2012-10-17\",
+            \"Statement\": [
+                {
+                    \"Action\": \"sts:AssumeRole\",
+                    \"Effect\": \"Allow\",
+                    \"Principal\": {
+                        \"Service\": \"lambda.amazonaws.com\"
+                    }
+                }
+            ]
+        }"
+        `
+
+        `aws iam create-role --role-name <YOUR_FSS_LAMBDA_EMAIL_ROLE> --assume-role-policy-document "${LAMBDA_TRUST}"`
+
+        where `<YOUR_FSS_LAMBDA_EMAIL_ROLE>` is replaced with the name you want to give to the role. Example: `FSS_Lambda_Email_Role`.
+    2. Attach the `AWSLambdaBasicExecutionRole` managed policy to the role:
+
+        `aws iam attach-role-policy --role-name FSS_Lambda_Email_Role --policy-arn arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole`
+
+    3. Attach the custom policy to the role:
+
+        `aws iam attach-role-policy --role-name FSS_Lambda_Email_Role --policy-arn <YOUR_FSS_LAMBDA_EMAIL_POLICY_ARN>`
+
+        where `<YOUR_FSS_LAMBDA_EMAIL_POLICY_ARN>` is replaced with the File Storage Security custom policy's ARN that you noted earlier. Example: `arn:aws:iam::0123456789012:policy/FSS_SES_Send_Email_Policy`.
+    </details>
+
+## Deploy the Lambda
+
+<details>
+<summary>Using the AWS console</summary>
+
+1. **Create an empty function**
+    - Go to **Services > Lambda**.
+    - Click **Create function**.
+    - Select the **Author from scratch** box.
+    - In the **Function name** field, enter a name. Example: `FSS_Scan_Send_Email`.
+    - From the **Runtime** drop-down list, select **Python 3.8**.
+    - Under **Permissions**, expand **Choose or create an execution role**.
+    - Select **Use an existing role**.
+    - In the drop-down list, select the execution role you created earlier. Example: `FSS_Lambda_Email_Role`.
+    - Click **Create function** and leave the page open.
+2. **Add function code**
+    - Download the Lambda function [handler.py file from GitHub](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/post-scan-actions/aws-python-email-notification/handler.py).
+    - On the AWS console page you left open, in the **Function code** section, remove the sample Lambda function code and paste the code from `handler.py`.
+    - Click **Save** at leave the page open.
```

Comment:
- Click **Save** and leave the page open.

Reply:
Ok

Reply:
Changed